src/material.cpp

```diff
@@ -60,8 +60,8 @@ namespace {
 
   // QueenMinorsImbalance[opp_minor_count] is applied when only one side has a queen.
   // It contains a bonus/malus for the side with the queen.
-  const int QueenMinorsImbalance[16] = { 
-    31, -8, -15, -25, -5, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
+  const int QueenMinorsImbalance[PIECE_NB] = {
```

Comment:
If we want to be able to access index number 12, it has to be at least QueenMinorsImbalance[13]...